### PR TITLE
Bump to 8.5.1

### PR DESCRIPTION
**Pull request by jsanz**
Bumping after tagging and releasing `8.5.0`.

Upgrading to a patch given there are no big changes here planned. We can always cut a `8.5` branch later if we want a minor release. This also saves maintenance overhead with backports, fixing dependencies and vulnerabilities in multiple branches and so on. 

Happy to discuss this, of course, if we feel better cutting `8.5` now.

### CODE REVIEW

**Review by nickpeihl — APPROVED** [review]
lgtm after #194 is merged. a patch release is fine for this doc update

**Comment by jsanz** [comment]
Oh sorry, I don't mean to do a release **now**, just bumping `master` to `8.5.1` to start a new entry in the changelog. 

It may happen in the future we actually need to bump to `8.6.0` if the changes accumulated justify it. Sorry if this is confusing.

Makes sense?

**Comment by nickpeihl** [comment]
Oh right. Yeah, makes sense. 😅

**Pull request merged**
